deno.json

```diff
@@ -8,7 +8,7 @@
   },
   "importMap": "./import_map.json",
   "tasks": {
-    "test": "deno test --unstable-http --unstable-webgpu --doc --allow-all --parallel --coverage --trace-leaks --clean",
+    "test": "deno test --unstable-http --unstable-webgpu --unstable-unsafe-proto --doc --allow-all --parallel --coverage --trace-leaks --clean",
```

Comment:
I don't think we should change this for the entire test run.

Can you add another task which only run `deno test --unstabel-unsafe-proto assert/object_match_test.ts`?

Reply:
Is there any reason not to enable this option for the entire tests? Deno std library might be used everywhere out of control of Deno, such as browsers. If such runtimes are included in targets of the std, disabling unsafe proto is meaningless.

IMHO I even think this option should be enabled by default for `deno test` CLI. Without this option, users can't notice there might be potential bugs and vulnerabilities that could happen only when run in other runtimes. This would be quite hard to debug.

Reply:
> Is there any reason not to enable this option for the entire tests? Deno std library might be used everywhere out of control of Deno, such as browsers. If such runtimes are included in targets of the std, disabling unsafe proto is meaningless.

Deno is currently the primary target of Deno std. The test cases should pass with the default state of Deno.

I think the compatibility with other runtimes should be checked by running test cases in those runtime, not simulating them in Deno.

> IMHO I even think this option should be enabled by default for deno test CLI. 

Open an issue in denoland/deno repository.